app/views/account/shared/_header.html.slim

```diff
@@ -0,0 +1,95 @@
+nav.header-navbar.navbar-expand-md.navbar.navbar-with-menu.navbar-static-top.navbar-dark.bg-gradient-x-grey-blue.navbar-border.navbar-brand-center
+  .navbar-wrapper
+    .navbar-header
+      ul.nav.navbar-nav.flex-row
+        li.nav-item.mobile-menu.d-md-none.mr-auto
+          a.nav-link.nav-menu-main.menu-toggle.hidden-xs[href="#"]
+            i.ft-menu.font-large-1
+        li.nav-item
+          = link_to(root_path, class: "navbar-brand") do
```

Comment:
here navigation need to go to dashboard url, change from root_url

Reply:
Thanks, fixed.